src/options/modules/ColorSchemeModeHelper.js

```diff
@@ -0,0 +1,16 @@
+/* this function changes screenshot in the options menu as per user's OS theme.By defualt 
+a user will see light themed screenshot but if the user has enabled dark theme
+in their OS the they would see dark-themed screenshot. It makes use of the following CSS media query
+and detects its change of state by attaching a js listener to it.
```

Comment:
This comment is not JSDoc. Please document the function (too), it has a specific format. (just look it up, there are likely also plugins)